src/hub/filters.py

```diff
@@ -40,7 +40,7 @@ def filter(self, qs, value):
                 + F("paper_count")
             )
 
-            qs = qs.annotate(score=score).order_by("-score")
+            qs = qs.annotate(score=score).order_by(*value, "id")
```

Comment:
so we're ordering first by score and then by id?

Reply:
`id` is meant to be a tie-breaker if the scores are the same. This change is to resolve the issue we had with the same item showing up on multiple pages.